beacon_node/client/src/builder.rs

```diff
@@ -967,6 +968,10 @@ where
                 runtime_context.executor.clone(),
                 beacon_chain.clone(),
             );
+            start_engine_version_cache_refresh_service(
```

Comment:
Is it likely that the `engine_version` is going to be used for other reasons?

If not, it would be nice if we could disable this service in the event that the Graffiti is defined by the BN since in that case we would always ignore the engine version entirely.

Reply:
Nice catch! I added a condition inside this method to disable the service if the graffiti is user-specified.